app/src/main/res/values/styles_v2.xml

```diff
@@ -728,6 +728,16 @@
     <item name="android:paddingStart">@dimen/spacing_16</item>
     <item name="android:paddingTop">@dimen/spacing_16</item>
   </style>
+
+  <style name="Clinic.V2.HistoryScreen.CardView" parent="Clinic.V2.CardView">
```

Comment:
I feel that layout attributes like margin, width, and height are specific to the screen the component is used in and hence should be part of the attributes in the parent screen. Whereas things like padding, background, etc are fine to place in the styles since they are common to the component no matter which screen it is placed in.

What do you think?

Reply:
Moved margin, height and width specific attributes to layout